README.md

```diff
@@ -158,7 +156,7 @@ containing all the features. You will also need to make sure to call `RUN config
 You can also set it through Dockerfile
 
 ```dockerfile
-FROM websphere-liberty:kernel
+FROM websphere-liberty:kernel-java8-openj9-ubi
```

Comment:
just need to add the `ibmcom/` prefix (for the other ones too)

Reply:
Oh, good catch. Thank you